pkgs/by-name/zo/zoho-mail/package.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, appimageTools
+, fetchurl
+
+}:
+appimageTools.wrapType2  rec  {
+  pname = "zoho-mail";
+  version = "1.6.1";
+
```

Comment:
add section `extrapkgs` to resolve dependencies of this package.

Reply:
To my understanding, appimages already bundle dependencies with it?

Reply:
Appimages are supposed to have everything they depend on bundled together, but from when I built `clickup` for Nix, I has to include a bunch of stuff to make it actually work with Nix. This was my extrapkgs line when building Clickup from the appimage. To my understanding, the dependencies bundled with the appimage will not work (at all) unless the dependency itself is present in the Nix Environment. For me, clickup failed to load due to a missing shared object from `libxkb`. 
```
extraPkgs = pkgs: with pkgs; [ xorg.libxkbfile alsa-lib dbus-glib gtk3 nss gnused libdbusmenu-gtk3 ];
```
P.S. I copied the entire package.nix file from someone over on reddit.

Reply:
According to the manual, it's to pass in runtime dependencies needed for FHS. You don't need to pass in dependencies already bundled in the appimage, but for stuff that it assumes is already on the system. What dependencies did you add to extrapkgs for this to work? Also, I added the appimage as a fallback - I want to focus on packaging the deb.

Reply:
Since I copied the package.nix from my impementation of clickup, I never bothered to change the extraPkgs from the `extraPkgs = pkgs: with pkgs; [ xorg.libxkbfile alsa-lib dbus-glib gtk3 nss gnused libdbusmenu-gtk3 ];` that the clickup's package.nix had.